docs/products/kafka/howto/enable-governance.md

```diff
@@ -0,0 +1,107 @@
+---
+title: Enable governance for Aiven for Apache Kafka®
+sidebar_label: Enable governance
+limited: true
+---
+import ConsoleLabel from "@site/src/components/ConsoleIcons"
+
+Enable governance in Aiven for Apache Kafka® to establish a secure and compliant framework to manage your Aiven for Apache Kafka clusters efficiently.
+
+## Prerequisites
+
+- This is a [limited availability feature](/docs/platform/concepts/beta_services). To try
+  it out, contact the sales team at [sales@aiven.io](mailto:sales@aiven.io).
+- You need [super admin](/docs/platform/howto/make-super-admin) permissions to
+  enable governance.
+
+## Enable governance
+
+1. Access the [Aiven Console](https://console.aiven.io/) and click **Admin**.
+1. On the organization page, click <ConsoleLabel name="Apache Kafka governance"/>.
+1. Click **Enable governance**.
+1. Select a user group to manage topics. This group is the default group and
+   owns all topics in the organization.
+   - If no user group exists, click **Create new user group**.
+     - Enter the group name and description.
+     - Click **Create group**.
+   - Select the created user group.
+1. Click **Next**.
+1. Define resource visibility for the Apache Kafka topic catalog.
+1. Select **Project** or **Organization** visibility.
+
+   - **Project**: Users can view Aiven for Apache Kafka resources in the Apache Kafka
+     topic catalog if they are project members.
+   - **Organization**: Users can view all Aiven for Apache Kafka resources across the
+     organization in the Apache Kafka topic catalog, regardless of their access to
+     individual projects.
+
+1. Click **Next**.
+1. Set global default topic configurations:
+   - To use the default settings, click **Keep defaults** and review the default
+     values on the confirmation window.
+   - Click **Enable governance**.
+
+   Alternatively, to customize:
+
+   - Enter your global default topic configurations.
+   - Click **Enable governance** to apply your changes.
```

Comment:
```suggestion
1. Set global default topic configurations.
   - To use the default settings:
     1. Click **Keep defaults** and review the default values on the confirmation window.
     1. Click **Enable governance**.
   - To customize the default settings:
     1. Enter your global default topic configurations.
     1. Click **Enable governance** to apply your changes.
```
?

Reply:
There are multiple ways to do the same operation. For now, I will keep this as is. I also looked up Google Style Guide for this.